### PR TITLE
update data (and html) so we don't count empty properties.

### PR DESCRIPTION
**Pull request by ndushay**
It turns out that counting empty values distorts our notion of our data.   Amazing.

### CODE REVIEW

File: data-no-catalog.csv [hunk 1]
```diff
@@ -1,158 +1,159 @@
+[DEPRECATION] Committee: please set parameter_overwite_by_rails_rule = false because we'll change default value to "true" in next major version.
```
Comment (jmartin-sul) [inline data-no-catalog.csv]: misdirected output?

File: data-catalog.csv [hunk 2]
```diff
@@ -1,177 +1,178 @@
+[DEPRECATION] Committee: please set parameter_overwite_by_rails_rule = false because we'll change default value to "true" in next major version.
```
Comment (jmartin-sul) [inline data-catalog.csv]: misdirected stdout/stderr?

File: data-all.csv [hunk 3]
```diff
@@ -1,233 +1,234 @@
+[DEPRECATION] Committee: please set parameter_overwite_by_rails_rule = false because we'll change default value to "true" in next major version.
```
Comment (jmartin-sul) [inline data-all.csv]: misdirected output?

**Comment by ndushay** [comment]
tagging @edsu so he sees this, even if it is merged first.  (no need to wait for edsu to merge it)

**Review by jmartin-sul — APPROVED** [review]
the CSV files appear to have captured a bit of unintended output, but otherwise ✅

File: data-all.csv (hunk 3 above)
Comment (ndushay) [reply data-all.csv]: yeah.  I'll remove.  Thanks for seeing it.

**Pull request merged**